src/index.tsx

```diff
@@ -100,4 +101,5 @@ export {
   Mixins,
   Range,
   SmallInput,
+  Themes,
```

Comment:
Non-blocking: should we export the Styled-components' ThemeProvider to allow the theme customization without requiring the application to have Styled-components as a dependency?